src/components/hardware-wallets/Layout.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { Box, Inset, Row, Rows } from '@/design-system';
+
+export function Layout({
+  children,
+  header,
+  footer,
+}: {
+  children: React.ReactNode;
```

Comment:
Probably `React.Children`? It's kinda OCP to not enforce children type

Reply:
yep yep, sounds good